src/main/java/net/datafaker/providers/videogame/MarvelSnap.java

```diff
@@ -0,0 +1,26 @@
+package net.datafaker.providers.videogame;
+
+import net.datafaker.providers.base.AbstractProvider;
+
+public class MarvelSnap extends AbstractProvider<VideoGameProviders> {
+
+    protected MarvelSnap(final VideoGameProviders faker) {
+        super(faker);
+    }
+
+    public String characters() {
```

Comment:
I think it's better to use the singular form for the getter, as it returns a single element.
For example : 
```
String character = faker.marvelSnap().character();
```

Reply:
Sure thing.

Reply:
This doesn't look too be resolved yet.